NEWS.md

```diff
@@ -32,13 +32,34 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
   intelmq.bots.experts.tor_nodes.expert --update-database
   ```
   The provided shell scripts use these new commands, however they are now deprecated and will be removed in version 3.0.
+- `intelmqctl list queues` is now able to count events in a queue with the `--sum`, `--count` or simple `-s` flag.
 
 #### XMPP Bots
 Both the XMPP output bot and the XMPP collector bot are deprecated.
 The bots need to be migrate to another XMPP library (see [#1614](https://github.com/certtools/intelmq/issues/1614) for details) and a survey on the mailing listed revealed no users.
 If you depend on this bot, please reach out to us via the mailing list or GitHub.
 The bots are logging a deprecation warning now and the current plan is to remove them in IntelMQ version 3.0.
 
+#### ShadowServer Bots
+ShadowServer Collector & Parser reworked. CSV and JSON reports are now parseable.
```

Comment:
Should better indicate that the JSON support is the new thing here, additionally to the existing CSV parser.
And the Collector and the parser have not been reworked. We added the JSON parsing functionality and a new collector for the Shadowserver Reports API.
Please also refer to the Bots documentation for more details.